Presentation.Web/app/models/ui-customization/config/blueprints/it-system-usage-ui-customization-config-blueprint.ts

```diff
@@ -8,7 +8,16 @@
             frontPage: {
                 text: "Systemforside",
                 readOnly: true,
-                helpText: Configs.helpTexts.cannotChangeTab
+                helpText: Configs.helpTexts.cannotChangeTab,
+                children: {
+                    dateFields: {
```

Comment:
not dataFields --> usagePeriod... datefields is also a bad word when used in danish bit it is what they want... we should however keep our wording closer to the actual meaning of the data